pkg/tui/verbosity.go

```diff
@@ -27,14 +27,8 @@ func (v Verbosity) LogLevel() zapcore.Level {
 	}
 }
 
-// CombineLogs controls whether to show all logs comingled in the TUI. In other words,
-// sorted by timestamp, not grouped by construct.
+// CombineLogs controls whether to show all errors commingled in the TUI.
+// In other words, sorted by timestamp, not grouped by construct.
```

Comment:
I don't think this comment is accurate - it's still just the logs comingled (uses `program.Println`) or grouped (uses `tui.model`).

Reply:
*sigh* Intellij refactoring strikes again.